src/oas2/transformers/__tests__/responses.test.ts

```diff
@@ -94,4 +95,75 @@ describe('responses', () => {
       expect(responses[0].contents![0].examples).toHaveLength(2);
     });
   });
+
+  describe('schema examples', () => {
+    it('given schema examples should translate to responses', () => {
+      const responses = translateToResponses(
+        {
+          r1: {
+            description: 'd1',
+            headers: {},
+            schema: {
+              example: {
+                name: 'value',
+              },
+            },
+          },
+        },
+        produces,
+      );
+      expect(responses[0].contents![0]).toHaveProperty('examples', [{ key: 'default', value: { name: 'value' } }]);
+    });
+
+    it('given multiple schema example properties should pick one', () => {
+      const responses = translateToResponses(
+        {
+          r1: {
+            description: 'd1',
+            headers: {},
+            schema: {
+              example: {
+                name: ' example value',
+              },
+              ['x-examples']: {
+                'application/json': {
+                  name: 'examples value',
+                },
+              },
+            } as Schema,
+          },
+        },
+        produces,
+      );
+      expect(responses[0].contents![0]).toHaveProperty('examples', [
+        { key: 'application/json', value: { name: 'examples value' } },
```

Comment:
I am not so sure about this, why are we ignoring the other examples?

Reply:
That's how we did it so far for standalone JSON Schema files https://github.com/stoplightio/elements/blob/beta/src/components/Docs/HttpOperation/utils.ts#L15
Also that was the idea in initial issue:
> Note that I think we only need to choose one at a time, in this priority order: examples, x-examples, example.

Though we can always combine them if you think it's better.

Reply:
Yes, we should combine them @mallachari — we should never lose data during the transformation.

Reply:
Alright, schema object examples are combined.